bindings/python/test/units/derived/test_angle.py

```diff
@@ -15,3 +19,337 @@ def test_angle_conversion():
+    assert angle.get_unit() == Unit.Arcsecond
+    assert angle.in_arcseconds() == 60.5
+
+    # Revolutions
+    angle: AnglePhysics = AnglePhysics.revolutions(3.9)
+
+    assert angle is not None
+    assert isinstance(angle, AnglePhysics)
+    assert angle.is_defined()
+    assert angle.get_unit() == Unit.Revolution
+    assert angle.in_revolutions() == 3.9
+
+
+def test_angle_unit_conversion():
+    angle: AnglePhysics = AnglePhysics(60.0, Unit.Degree)
+
+    assert angle.in_degrees() == 60.0
+    assert angle.in_radians() == (math.pi * 60.0 / 180.0)
+    assert angle.in_arcminutes() == 60.0 * 60.0
+    assert (angle.in_arcseconds() - 60.0 * 3600.0) <= 1e-10
+
+
+def test_angle_comparators():
+    angle_deg: AnglePhysics = AnglePhysics(60.0, Unit.Degree)
+    angle_rad: AnglePhysics = AnglePhysics(60.0 * math.pi / 180.0, Unit.Radian)
+
+    assert angle_deg == angle_deg
+    assert angle_rad == angle_rad
+    assert angle_deg == angle_rad
+    assert angle_deg != AnglePhysics(59.0, Unit.Degree)
+
+
+def test_angle_operators():
+    angle: AnglePhysics = AnglePhysics(30.0, Unit.Degree)
+    half_angle: AnglePhysics = angle / 2.0
+    double_angle: AnglePhysics = angle * 2.0
+    sum_angle: AnglePhysics = angle + angle
+    diff_angle: AnglePhysics = angle - angle
+    positive_angle: AnglePhysics = +angle
+    negative_angle: AnglePhysics = -angle
+
+    assert half_angle is not None
+    assert isinstance(half_angle, AnglePhysics)
+    assert double_angle is not None
+    assert isinstance(double_angle, AnglePhysics)
+    assert sum_angle is not None
+    assert isinstance(sum_angle, AnglePhysics)
+    assert diff_angle is not None
+    assert isinstance(diff_angle, AnglePhysics)
+    assert positive_angle is not None
+    assert isinstance(positive_angle, AnglePhysics)
+    assert negative_angle is not None
+    assert isinstance(negative_angle, AnglePhysics)
+
+    assert half_angle.in_degrees() == 15.0
+    assert double_angle.in_degrees() == 60.0
+    assert sum_angle.in_degrees() == 60.0
+    assert diff_angle.in_degrees() == 0.0
+    assert diff_angle.is_zero()
+    assert positive_angle.in_degrees() == +30.0
+    assert negative_angle.in_degrees() == -30.0
+
+    angle_deg: AnglePhysics = AnglePhysics(90.0, Unit.Degree)
+    angle_rad: AnglePhysics = AnglePhysics(math.pi / 2, Unit.Radian)
+
+    sum_angle: AnglePhysics = angle_rad + angle_deg
+    assert sum_angle is not None
+    assert isinstance(sum_angle, AnglePhysics)
+    assert (sum_angle.in_degrees() - 180.0) <= 1e-10
+
+    angle: AnglePhysics = AnglePhysics(90.0, Unit.Degree)
+
+    angle *= 3.0
+    assert angle is not None
+    assert isinstance(angle, AnglePhysics)
+    assert angle.get_unit() == Unit.Degree
+    assert angle.in_degrees() == 270.0
+
+    angle /= 3.0
+    assert angle is not None
+    assert isinstance(angle, AnglePhysics)
+    assert angle.get_unit() == Unit.Degree
+    assert angle.in_degrees() == 90.0
+
+
+# def test_angle_between_vector2d ():
```

Comment:
Should these 3 comments be removed?

Reply:
This is none implemented pieces of code. Let's clean and remove I'd say

Reply:
done